src/networks.jl

```diff
@@ -99,28 +108,50 @@ end
 # automatically detect type parameter:
 NodalMachine(model::M, args...) where M<:Model = NodalMachine{M}(model, args...)
 
-# to turn fit-through fitting off and on:
+"""
+$SIGNATURES
+
+Turn fit-through off for a given `NodalMachine`.
+See also [`thaw!`](@ref).
+"""
 function freeze!(machine::NodalMachine)
```

Comment:
Maybe better is: "Freeze the machine `machine`, so that it will never be retrained."